corehq/apps/app_manager/views/download.py

```diff
@@ -283,16 +284,19 @@ def create_build_files_if_necessary_handling_conflicts(is_retry=False):
             request.app = Application.get(request.app.get_id)
             create_build_files_if_necessary_handling_conflicts(True)
 
+    # Todo; remove after https://dimagi-dev.atlassian.net/browse/ICDS-1483 is fixed
```

Comment:
might be worth taking a note in the ticket to remember this.

Reply:
Yes, it's tracked.